mkosi/resources/mkosi.md

```diff
@@ -2023,6 +2029,13 @@ current working directory. The following scripts are supported:
   artifacts from `SplitArtifacts=yes` or RPMs built in a build script).
   Note that this script does not use the tools tree even if one is configured.
 
+* If **`mkosi.postoutput`** (`PostOutputScripts=`) exists, it is executed right
```

Comment:
Nit: this point should maybe go above the `mosi.clean` script, since logically cleaning things will be a thing that happens after having generated outputs.

Reply:
This also needs a column in the table for environment variables below.